gradle.properties

```diff
@@ -26,7 +26,7 @@ spotbugsAnnotationsVersion=3.1.12
 spotbugsPluginVersion=1.6.9
 jcipAnnotationsVersion=1.0
 gradleDockerPlugin=3.1.0
-gradleLombokPluginVersion=1.14
+gradleLombokPluginVersion=3.2.0
```

Comment:
This is quite a big version increment. Are we sure we won't introduce unwanted problems by this?

Reply:
Reviewed the release notes and do not observe any issues. Ran the javadocs gradle task and do not observe issues.